scm-core/src/main/java/sonia/scm/repository/work/SimpleWorkingCopyFactory.java

```diff
@@ -100,11 +103,18 @@
   private static final Logger LOG = LoggerFactory.getLogger(SimpleWorkingCopyFactory.class);
 
   private final WorkingCopyPool workingCopyPool;
+  private MeterRegistry meterRegistry;
```

Comment:
Please make the field `final` and add a `javax.annotation.Nullable` annotation to it.

Reply:
Another option could be the usage of an empty MeterRegistry instead of `null`.